### PR TITLE
Features/reviews in profile

### PR DESCRIPTION
**Pull request by lenchv**

### CODE REVIEW

File: resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            this.props.setGivenReviews(response.data);
+        }).catch(() => this.setState({preloader: false}));
+    }
+
+    render() {
+        const {givenReviews} = this.props,
+            {preloader} = this.state;
+
+        return (
+            <div style={{position: 'relative'}}>
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js]: avoid using inline styles

File: resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <Preloader enable={preloader} />
+                <ReviewsListEmpty show={givenReviews.length <= 0 && !preloader} />
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js]: avoid using such statements, define variable or method that will describe this logic

File: resources/assets/js/features/user/components/Profile/Reviews/ReceivedContainer.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                this.setState({preloader: false});
+            });
+    }
+
+    render() {
+        const {rating, receivedReviews} = this.props,
+            {preloader} = this.state;
+
+        return (
+            <div style={{position: 'relative'}}>
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/ReceivedContainer.js]: avoid using inline styles

File: resources/assets/js/features/user/components/Profile/Reviews/ReceivedContainer.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                <Preloader enable={preloader} />
+                <Rating marks={rating} />
+                <ReviewsList list={receivedReviews}/>
+                <ReviewsListEmpty show={receivedReviews.length <= 0 && !preloader} />
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/ReceivedContainer.js]: avoid using such statements in template, define variable or method that will return appropriate value

File: resources/assets/js/features/user/components/Profile/Reviews/Review.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            <div className="row comment">
+                <div className="col-sm-2">
+                    <img className="img-responsive user-photo" src={user.photo ? user.photo : defaultUserPhoto}/>
+                </div>
+                <div className="col-sm-10">
+                    <div className="card card-comment">
+                        <div className="card-header comment-header">
+                            <strong>{user.full_name}</strong>&nbsp;
+                            <span className="text-muted">
+                                {(Number.isInteger(commentDate) && commentDate <= 7)
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/Review.js]: avoid using such statements in templates, define variable with appropriate value or method that will return needed value

File: resources/assets/js/features/user/components/Profile/Reviews/ReviewsList.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        const {list, reviews, users} = this.props;
+
+        return (
+            <div>
+                {list.map((id) => (
+                    <Review
+                        key={id}
+                        mark={reviews.byId[id].mark}
+                        date={reviews.byId[id].date * 1000}
+                        user={users.byId[reviews.byId[id].user]}
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/ReviewsList.js]: create method that will know how to find user from given params

File: resources/assets/js/features/user/components/Profile/Reviews/ReviewsListEmpty.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ReviewsListEmpty extends React.Component {
+    render() {
+        const {show} = this.props,
+            displayStyle = show ? {} : {display: 'none'};
+
+        return (
+            <div style={displayStyle}>
```
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/ReviewsListEmpty.js]: avoid using inline styles

File: resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js (hunk 2 above)
Comment (TatsuUkraine) [inline resources/assets/js/features/user/components/Profile/Reviews/GivenContainer.js]: render ReviewsListEmpty or ReviewsList based on some criteria, not both in same time

**Review by TatsuUkraine — CHANGES_REQUESTED** [review]

**Review by TatsuUkraine — APPROVED** [review]

**Review by sergey-ben — APPROVED** [review]

**Pull request merged**